com/DRA.py

```diff
@@ -218,29 +244,36 @@ def callback_handler(job):
                 with open('%s_hist.png' % job.result[5], "wb") as fh:
                     fh.write(base64.b64decode(job.result[4]))
         else:
-            tqdm.write(' -- Station %s.%s did not produce valid statistics' % (job.result[6], job.result[7]))
+            tqdm.write(' -- Station %s.%s did not produce valid statistics' % (
+                job.result[6], job.result[7]))
 
 
 def main():
 
     global stn_stats, wrms_n, wrms_e, wrms_u, project
 
-    parser = argparse.ArgumentParser(description='GNSS daily repetitivities analysis (DRA)')
+    parser = argparse.ArgumentParser(
+        description='GNSS daily repetitivities analysis (DRA)')
 
     parser.add_argument('project', type=str, nargs=1, metavar='{project name}',
-                        help="Specify the project name used to process the GAMIT solutions in Parallel.GAMIT.")
+                        help='''Specify the project name used to process
```

Comment:
I also prefer the triple quotes for handling text blocks. This LGM, test pass-- thanks for the PR